$MirOS: ports/plan9/kencc/patches/patch-utils_ql_l_h,v 1.1 2006/03/31 03:50:24 tg Exp $
--- utils/ql/l.h.orig	Thu Mar 30 23:34:31 2006
+++ utils/ql/l.h	Thu Mar 30 23:55:25 2006
@@ -282,7 +282,7 @@ void	doprof1(void);
 void	doprof2(void);
 void	dynreloc(Sym*, long, int, int, int);
 long	entryvalue(void);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 void	exchange(Prog*);
 void	export(void);
 int	find1(long, int);
@@ -305,7 +305,7 @@ void	nocache(Prog*);
 void	noops(void);
 void	nuxiinit(void);
 void	objfile(char*);
-int	ocmp(void*, void*);
+int	ocmp(const void*, const void*);
 long	opcode(int);
 Optab*	oplook(Prog*);
 void	patch(void);
